activerecord/test/cases/adapters/sqlite3/sqlite3_adapter_test.rb

```diff
@@ -625,6 +625,16 @@ def test_doesnt_error_when_a_select_query_is_called_while_preventing_writes
         end
       end
 
+      def test_doesnt_error_when_a_read_query_with_leading_chars_is_called_while_preventing_writes
+        with_example_table "id int, data string" do
+          @conn.execute("INSERT INTO ex (data) VALUES ('138853948594')")
+
+          @conn.while_preventing_writes do
+            assert_equal 1, @conn.execute("  SELECT data from ex WHERE data = '138853948594'").count
```

Comment:
I didn't add leading parenthesis to the sqlite3 test because sqlite3 didn't consider it valid syntax.